build/Targets/Imports.targets

```diff
@@ -470,4 +471,23 @@
 
   <!-- Returns the current build version. Used in .vsixmanifests to substitute our build version into them -->
   <Target Name="GetBuildVersion" Outputs="$(VsixVersion)" />
+
+  <!-- 
+    In order to leverage LUT testing we need to have both of the follownig packages in 
+    our build.  
+
+        - xunit.runner.desktop
+        - xunit
+
+    Both of these include xunit.abstractions.dll in different forms: content and reference
+    respectively.  This creates a double write violation in our build.  The content item
+    is unnecessary for us since we reference the xunit package hence remove it here. 
+
+    https://github.com/dotnet/roslyn/issues/18753
+  -->
+  <Target Name="RemoveDuplicateXUnitContent">
+    <ItemGroup>
+      <Content Remove="@(Content)" Condition="'%(Filename)%(Extension)' == 'xunit.abstractions.dll'" />
```

Comment:
@jasonmalinowski am curious to get your feedback on this.  Do I need to add Inputs and Outputs to this target to be correct?